src/main/java/com/google/cloud/tools/libraries/CloudLibraries.java

```diff
@@ -21,12 +21,14 @@
 import com.google.gson.Gson;
 import com.google.gson.reflect.TypeToken;
 import com.google.gson.stream.JsonReader;
+
```

Comment:
This should be reverted per Google style. Hope you don't get a warning.

Reply:
done